ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-release-3.74-presubmits.yaml

```diff
@@ -1260,6 +1260,162 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )ocp-4-12-qa-e2e-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I think it's fine to keep these new presubmits as `always_run: true` even though the job might be failing.

Reply:
I'm going to change this to make it consistent with this discussion: https://github.com/openshift/release/pull/38578#discussion_r1175588476